src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/PublicationController.cs

```diff
@@ -17,98 +17,103 @@
-                .HandleFailuresOrOk();
-        }
+    [MemoryCache(typeof(ListPublicationsPostCacheKey), durationInSeconds: 10, expiryScheduleCron: HalfHourlyExpirySchedule)]
+    [HttpPost("publications")]
+    public async Task<ActionResult<PaginatedListViewModel<PublicationSearchResultViewModel>>> ListPublications(
+        [FromBody] PublicationsListPostRequest request)
+    {
+        return await _publicationService
+            .ListPublications(
+                request.ReleaseType,
+                request.ThemeId,
+                request.Search,
+                request.Sort,
+                request.SortDirection,
+                page: request.Page,
+                pageSize: request.PageSize,
+                publicationIds: request.PublicationIds)
+            .HandleFailuresOrOk();
+    }
 
-        [HttpGet("publications/{slug}/title")]
-        public async Task<ActionResult<PublicationTitleViewModel>> GetPublicationTitle(string slug)
-        {
-            return await _publicationCacheService.GetPublication(slug)
-                .OnSuccess(p => new PublicationTitleViewModel
-                {
-                    Id = p.Id,
-                    Title = p.Title,
-                })
-                .HandleFailuresOrOk();
-        }
+    [HttpGet("publications/{publicationId:guid}/summary")]
+    public async Task<ActionResult<PublishedPublicationSummaryViewModel>> GetPublicationSummary(Guid publicationId)
+    {
+        return await _publicationService
+            .GetSummary(publicationId)
+            .HandleFailuresOrOk();
+    }
 
-        [HttpGet("publications/sitemap-items")]
-        public async Task<ActionResult<List<PublicationSitemapItemViewModel>>> ListSitemapItems(
-            CancellationToken cancellationToken = default) =>
-            await _publicationService.ListSitemapItems(cancellationToken)
-                .HandleFailuresOrOk();
+    [HttpGet("publications/{slug}/title")]
+    public async Task<ActionResult<PublicationTitleViewModel>> GetPublicationTitle(string slug)
+    {
+        return await _publicationCacheService.GetPublication(slug)
+            .OnSuccess(p => new PublicationTitleViewModel
+            {
+                Id = p.Id,
+                Title = p.Title,
+            })
+            .HandleFailuresOrOk();
     }
+
+    [HttpGet("publications/sitemap-items")]
+    public async Task<ActionResult<List<PublicationSitemapItemViewModel>>> ListSitemapItems(
+        CancellationToken cancellationToken = default) =>
+        await _publicationService.ListSitemapItems(cancellationToken)
+            .HandleFailuresOrOk();
+    
+    [HttpGet("publicationInfos")]
+    public async Task<ActionResult<IList<PublicationInfoViewModel>>> ListPublicationInfos(
+        [FromQuery] GetPublicationInfosRequest request, 
+        CancellationToken cancellationToken = default) =>
+        Ok(await _publicationService.ListPublicationInfos(request.ThemeId, cancellationToken));
```

Comment:
Looks like there are bunch of changes on this PR like this one and changes in QueryableExtensions, ArrayExtensions, GuidExtensions etc that already got merged to the dev branch by https://github.com/dfe-analytical-services/explore-education-statistics/pull/6036 so I'm unsure why they are showing in the diff of this PR. Could be because you reused the same branch?